var/spack/repos/builtin/packages/of-adios-write/package.py

```diff
@@ -0,0 +1,136 @@
+##############################################################################
+# Copyright (c) 2017 Mark Olesen, OpenCFD Ltd.
+#
+# This file was authored by Mark Olesen <mark.olesen@esi-group.com>
+# and is released as part of spack under the LGPL license.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for the LLNL notice and LGPL.
+#
+# License
+# -------
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+#
+# Legal Notice
+# ------------
+# OPENFOAM is a trademark owned by OpenCFD Ltd
+# (producer and distributor of the OpenFOAM software via www.openfoam.com).
+# The trademark information must remain visible and unadulterated in this
+# file and via the "spack info" and comply with the term set by
+# http://openfoam.com/legal/trademark-policy.php
+#
+# This file is not part of OpenFOAM, nor does it constitute a component of an
+# OpenFOAM distribution.
+#
+##############################################################################
+from spack import *
+from spack.environment import *
+import llnl.util.tty as tty
+
+import os
+from spack.pkg.builtin.openfoam_com import *
+
+
+class OfAdiosWrite(Package):
+    """adios-write supplies additional libraries and function objects
+    for reading/writing OpenFOAM data with ADIOS.
+    This offering is part of the community repository supported by OpenCFD Ltd,
+    producer and distributor of the OpenFOAM software via www.openfoam.com,
+    and owner of the OPENFOAM trademark.
+    OpenCFD Ltd has been developing and releasing OpenFOAM since its debut
+    in 2004.
+    """
+
+    # Currently only via git, but with some branches corresponding to main
+    # OpenFOAM releases.
+    homepage = "https://develop.openfoam.com/Community/feature-adiosWrite/"
+    gitrepo  = "https://develop.openfoam.com/Community/feature-adiosWrite.git"
+
+    version('plus', branch='develop', git=gitrepo)
```

Comment:
It would be better to call this version `develop` as I mentioned in your other PR.